packages/components/src/components/eventName/EventName.tsx

```diff
@@ -23,6 +30,12 @@ const EventName: React.FC<EventNameProps> = ({ event }) => {
         </span>
       )}
       {name}
+      {isHelsinkiCityOwned && (
+        <IconCheckCircleFill
```

Comment:
Should this have a tooltip to explain what it is?

Reply:
Same as in https://github.com/City-of-Helsinki/events-helsinki-monorepo/pull/448#discussion_r1323987877